src/main/xcomponents/styledComponents2/Button.jsx

```diff
@@ -0,0 +1,46 @@
+import styled from 'styled-components';
```

Comment:
Don't we already have a button component? Shouldn't we use that for consistency?

Reply:
styles in styleComponents are messed up, just take a look at Form.js is flex suppose to wrap all elements inside the RequestBar components? it does not make sense. That is why I am using css grid to have more control over the elements inside the form